tools/integration_tests/tokenserver/test_support.py

```diff
@@ -60,20 +71,65 @@ def tearDown(self):
 
         self.database.close()
 
-    def _forge_oauth_token(self, generation=None, sub='test', scope='scope'):
+    def _build_oauth_headers(self, generation=None, user='test',
```

Comment:
`_build_oauth_headers()` and `_build_browserid_headers()` are responsible for building the bogus auth headers I referenced [here](https://github.com/mozilla-services/syncstorage-rs/pull/1216/files#r799825107)

Reply:
That's way better :) My Python skills are rusty